publish_to_hugging_face.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import logging
+import sys
+
+from app_utils import hugging_face_utils
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="")
+    parser.add_argument(
+        "-e",
+        "--experiment_name",
+        required=True,
+        help="experiment name",
+        default=argparse.SUPPRESS,
+    )
+
+    parser.add_argument(
+        "-d",
+        "--device",
+        help="'cpu' or 'cuda:0', if the GPU device id is 0",
+        default="cuda:0",
+    )
+
+    parser.add_argument(
+        "-a",
+        "--api_key",
+        required=False,
+        help="Hugging Face API Key",
+        default=argparse.SUPPRESS,
+    )
+
+    parser.add_argument(
+        "-u",
+        "--user_id",
+        required=False,
+        help="Hugging Face User ID",
+        default=argparse.SUPPRESS,
+    )
+
+    parser.add_argument(
+        "-m",
+        "--model_name",
+        required=True,
```

Comment:
imho, this can be made optional with the suggestion that I made before

> model name on HF: apply hf_repo_friendly_name on the folder name of the model weights (only the last folder, not the full path) (optional custom name)

Reply:
Done